.pre-commit-config.yaml

```diff
@@ -8,16 +8,31 @@ repos:
   rev: v4.4.0
   hooks:
   - id: check-ast
-  - id: trailing-whitespace
-  - id: end-of-file-fixer
   - id: check-merge-conflict
     exclude: .*\.rst
-  - id: check-symlinks
   - id: check-case-conflict
+  - id: name-tests-test
+    args: [--pytest-test-first]
+    exclude: plasmapy/tests/_helpers/.*
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
   - id: check-json
   - id: check-toml
   - id: check-yaml
 
+- repo: https://github.com/sirosen/texthooks
+  rev: 0.5.0
+  hooks:
+  - id: fix-smartquotes
+  - id: fix-spaces
+  - id: fix-ligatures
+  - id: forbid-bidi-controls
```

Comment:
These are some changes to make our code more ASCII-like and avoid weird Unicode issues.  `fix-spaces` is to make sure that all spaces are just regular spaces (and not special Unicode spaces), and the last one is to prevent Unicode markers that indicate direction changes.